feature-libs/product/configurators/common/src/components/config-attribute/config-attribute-types/config-attribute-multi-selection-image/config-attribute-multi-selection-image.component.ts

```diff
@@ -8,22 +8,23 @@ import {
 } from '@angular/core';
 import { FormControl } from '@angular/forms';
 import { Configurator } from '@spartacus/core';
-import { ConfigFormUpdateEvent } from '../../config-form/config-form.event';
-import { ConfigUIKeyGeneratorService } from '../../service/config-ui-key-generator.service';
+import { ConfigFormUpdateEvent } from '../../../config-form/config-form.event';
+import { ConfigUIKeyGeneratorService } from '../../../service/config-ui-key-generator.service';
+import { ConfigUtilsService } from '../../../service/config-utils.service';
 @Component({
   selector: 'cx-config-attribute-multi-selection-image',
   templateUrl: './config-attribute-multi-selection-image.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ConfigAttributeMultiSelectionImageComponent implements OnInit {
-  constructor(public uiKeyGenerator: ConfigUIKeyGeneratorService) {}
-
   @Input() attribute: Configurator.Attribute;
   @Input() group: string;
   @Input() ownerKey: string;
 
   @Output() selectionChange = new EventEmitter<ConfigFormUpdateEvent>();
 
+  constructor(protected configUtilsService: ConfigUtilsService) {}
+
   attributeCheckBoxForms = new Array<FormControl>();
 
   ngOnInit() {
```

Comment:
Please specify a type for `let attributeCheckBoxForm`

Reply:
Done